# PosVel.msg

# This is a custom message representing the robot's position and velocity.


float64 x	# x-coordinate of the robot's position
float64 y	# y-coordinate of the robot's position
float64 vel_x	# linear velocity of the robot in the x-axis
float64 vel_z	# angular velocity of the robot in the z-axis
